src/FreeScribe.client/UI/SettingsWindow.py

```diff
@@ -123,8 +123,6 @@ def __init__(self):
             "Use best_of",
             "best_of",
             "Real Time Audio Length",
-            "Use Pre-Processing",
-            "Use Post-Processing",
```

Comment:
Instead of making them auto get listed, we manually did them above each textbox so this is OK